### ISSUE TITLE
2sxc-unpublish function should show no data to public users (instead of demo entity)

### ISSUE THREAD

**Issue by reto-f-999** [2sic/2sxc#1248]
**I'm submitting a ...**  
```
- [x] not sure
```

**...about**   
```
- [x] admin experience UI
- [x] Content Types or data management
```

**Current behavior**
When an admin uses the 2sxc-unpublish function on a normal 2sxc-module (1-item-only, no list functionality), the system shows the demo-entity to public users.

**Expected behavior**
When an admin uses the 2sxc-unpublish function..., the system should show nothing to public users, like there would be no module.

**Minimal reproduction of the problem with instructions**
1. add a 2sxc-module on any page
2. choose a 2sxc content type with a template, which is not using the 2sxc-list functionality
3. add any data to the entity and save it (published)
4. edit entity again, and do a unpublished save
5. logout to see what a public users gets

**What is the motivation / use case for changing the behavior?**
correct functionality - any admin expects this function to just hide (= unpublish) the data. 

**Please tell us about your environment:**
2sic environment

* **2sxc version(s):** 9.3.0 (official) 
* **Browser:** [all]
* **DNN:** [all]
* **Language:** [any/all]

I am posting this issue, because its still there after all the rework from past versions, and its a serious issue to normal admins, because in most cases they do not realize that they are showing the demo entity to public users. If they would know, they would rather delete the whole module instead.

**Comment by mick-bsi** [comment]
Hi @iJungleboy, any new information about that issue ? Thank you !

**Comment by iJungleboy** [comment]
No update yet. We're currently prioritizing 2sxc 9.7 - 9.10, after that we'll probably look into this. So no priority for another ca. 2 months at least.

**Comment by reto-f-999** [comment]
@mick-bsi workaround hint: if you set the demo item of the template to "unpublished", you get a result that is pretty close to the desired solution. 
(But be aware that this could lead to other unwanted side effects.)

**Comment by andyduo** [comment]
Did this get resolved at all? Running into this issue at the moment.

**Comment by iJungleboy** [comment]
@andyduo at the moment we're not working on this. There is always a simple workaround to use a demo-item with a unique property, and in your razor hiding that. To resolve this at the 2sxc level is very difficult, because some people want the demo to show, others do not.

**Comment by iJungleboy** [comment]
Important: this issue was moved/merged with #1792 which was just updated with an extensive discussion, please contribute your opinions.
